components/NavigationDrawer/src/MDCBottomDrawerViewController.m

```diff
@@ -57,6 +75,63 @@ - (void)setTrackingScrollView:(UIScrollView *)trackingScrollView {
   _transitionController.trackingScrollView = trackingScrollView;
 }
 
+- (void)setTopCornersRadius:(CGFloat)radius forDrawerState:(MDCBottomDrawerState)drawerState {
+  _topCornersRadius[@(drawerState)] = @(radius);
+
+  if (drawerState == MDCBottomDrawerStateCollapsed) {
+    _maskLayer.maximumCornerRadius = radius;
+  } else {
+    _maskLayer.minimumCornerRadius = [self getMinimumCornerRadius];
+  }
+}
+
+- (CGFloat)getMinimumCornerRadius {
```

Comment:
_Tip:_` - (CGFloat)minimumCornerRadius ` instead of `- (CGFloat)getMinimumCornerRadius`.